### ISSUE TITLE
window shadow and window corners with transparency

### ISSUE THREAD

**Issue by c-sanchez** [henrypp/freeshooter#15]
I'm not sure if the issue #13 was closed properly, I'm sorry if i'm disturbing with my ideas :P
Maybe my error by don't explain me a bit better heh.

Please check the 1st and 2nd images corners, from issue #13 both images are saved as PNG, so the transparency is possible here.
But freeshooter is not using the PNG transparency for window corners, if the imagen is saved as JPG or any other format without transparency, is ok, but if we have PNG as format to save images, I think the more proper is save the selected window using the transparency for the corners.
ie. ignoring the background and programs behind the taked window, after all we are taking a screenshot of the current window and nothing else.

By example, this is the result from take a screenshot of current window now
![Image](https://image.ibb.co/k88ybo/sshot_001.png)
is NOT a manual selection.
Here is capturing the current window, 'recycle bin', with the window shadow, yes, but also is capturing all behind the window, 'firefox window, firefox about window and desktop background' in this case.
ie, is not much different from making a manual capture, where if is logic take window, background, programs, etc

So, basically the issue is, if a window is captured, use the transparency for window corners, and if 'include windw shadow' option is enabled, capture the window and the shadow but with transparency (for png) wihtout any other window/program in the back, or the desktop background.

**Comment by henrypp** [comment]
> Here is capturing the current window, 'recycle bin', with the window shadow, yes, but also is capturing all behind the window, 'firefox window, firefox about window and desktop background' in this case.

This is not a bug, this is feature.

`Include window shadow` are add paddings to resulted sreenshot.
`Clear window background` are fill background behind captured window.

Sometimes i need window image with padding, but without filling window background. Sometimes not. Anyway two independed settings are worked as well.

PNG transparency is in my TODO.

**Comment by KrasnayaPloshchad** [comment]
Since WinSnap is already have this feature, it’s reasonable to implement too.
